modules/ROOT/pages/logging-mdc.adoc

```diff
@@ -18,7 +18,7 @@ To use the MDC Logging operations, complete the following tasks:
 * Change the pattern layouts in the `log4j2.xml` file to `MDC`.
```

Comment:
If you are using MDC Logging with Anypoint Runtime Fabric or CloudHub 2.0, you dont need to follow this step. Change the pattern layouts in the log4j2.xml file to MDC
```suggestion
* If you are using MDC Logging with Anypoint Runtime Fabric or CloudHub 2.0, you don't need to follow this pre-requisite. Otherwise, change the pattern layouts in the log4j2.xml file to MDC
```